README.rst

```diff
@@ -23,22 +24,117 @@ implementation are in the accepted
-build/installation.  This is similar to the ``ez_setup.py`` module that is
-shipped with some projects to bootstrap `setuptools
-<https://bitbucket.org/pypa/setuptools>`_.
+build/installation.
 
-As described in APE4, the version numbers for ``astropy_helpers`` follow the
-corresponding major/minor version of the `astropy core package
-<http://www.astropy.org/>`_, but with an independent sequence of micro (bugfix)
-version numbers. Hence, the initial release is 0.4, in parallel with Astropy
-v0.4, which will be the first version  of Astropy to use ``astropy-helpers``.
+As described in `APE4 <https://github.com/astropy/astropy-APEs/blob/master/APE4.rst>`_, the version
+numbers for ``astropy_helpers`` follow the corresponding major/minor version of
+the `astropy core package <http://www.astropy.org/>`_, but with an independent
+sequence of micro (bugfix) version numbers. Hence, the initial release is 0.4,
+in parallel with Astropy v0.4, which will be the first version  of Astropy to
+use ``astropy-helpers``.
 
 For examples of how to implement ``astropy-helpers`` in a project,
 see the ``setup.py`` and ``setup.cfg`` files of the
 `Affiliated package template <https://github.com/astropy/package-template>`_.
 
+What does astropy-helpers provide?
+----------------------------------
+
+Custom setup.py commands
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+The main part of astropy-helpers is to provide customized setuptools commands.
+For example, in a package that uses astropy-helpers, the following command
+will be available::
+
+    python setup.py build_docs
+
+and this command is implemented in astropy-helpers.
+
+The commands we provide or customize are:
+
+**python setup.py test**
+
+This command will automatically build the package, install it to a temporary
+directory, and run the tests using `pytest <http://pytest.org/>`_ on this
+installed version. Note that the bulk of this command is actually defined
+in ``astropy.tests.command.AstropyTest`` - here we simply define the custom
+setuptools command.
+
+**python setup.py sdist**
+
+We redefine ``sdist`` to use the version from distutils rather than from
+setuptools, as the setuptools version requires duplication of information
+in ``MANIFEST.in``.
+
+**python setup.py build_docs**
+
+This command will automatically build the package, then run sphinx to build
+the documentation. Sphinx itself provides a custom setuptools command, which we
+expand with the following options:
+
+* ``-w``: set the return code to 1 if there are any warnings during the build
+  process.
+
+* ``-l``: completely clean previous builds, including files generated by
+  the sphinx-automodapi package (which creates API pages for different
+  functions/classes).
+
+* ``-n``: disable the intersphinx option.
+
+* ``-o``: open the documentation in a browser if a build finishes successfully.
+
+In addition, build_docs will automatically download and temporarily install sphinx-astropy
```

Comment:
What does "temporarily install sphinx-astropy" mean?
Is it installed and removed again? In which environment / site-packages?
Does this always happen or only on first run?